framework/global/widgetstatestore.h

```diff
@@ -0,0 +1,32 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef WIDGETSTATESTORE_H
+#define WIDGETSTATESTORE_H
+
+class QWidget;
+
+class WidgetStateStore
```

Comment:
Why no `namespace mu` and `namespace framework` for this class?

Reply:
Now I did not add a namespace for simplicity, because this class is not used in `mu` classes.